Microsoft.Toolkit.Uwp.UI.Controls.Core/InAppNotification/InAppNotification.cs

```diff
@@ -34,7 +36,11 @@ public InAppNotification()
         {
             DefaultStyleKey = typeof(InAppNotification);
 
+            _dispatcherQueue = DispatcherQueue.GetForCurrentThread();
```

Comment:
Good call, I suppose with a UI control (compared to something like ThemeListener) there'd be no reason to use a DispatcherQueue other than the one we get initially on the UI Thread here (as the UI Thread should hopefully be the one initializing the control...).

Right @azchohfi?

Reply:
That's what I figured. Plus, XAML objects don't like non-empty constructors, and even declaring the default as null is not enough to appease the compiler. This made most sense to me :)